.github/workflows/pr-build-image.yaml

```diff
@@ -52,7 +52,26 @@ jobs:
           git fetch base-origin ${{ github.event.pull_request.base.ref }}
           git merge --no-edit base-origin/${{ github.event.pull_request.base.ref }}
 
+      - name: Determine Changed Files
+        id: changes
+        run: |
+          BASE_COMMIT=${{ github.event.pull_request.base.sha }}
+          HEAD_COMMIT=${{ github.event.pull_request.head.sha }}
+
+          CHANGED_FILES=$(git diff --name-only "$BASE_COMMIT" "$HEAD_COMMIT")
+
+          echo "Changed files:"
+          echo "$CHANGED_FILES"
+
+          if echo "$CHANGED_FILES" | grep -qv '^e2e-tests/'; then
```

Comment:
@gustavolira Would it be possible to add more checks(e.g. docs only change) if the code change is not required to build a new container image? we used to have more thorough checks in the old pipeline to avoid building a new image.  The logic and code look good though. Thanks!

Reply:
@josephca docs was always ignored. Please take a look at this line https://github.com/redhat-developer/rhdh/pull/2135/files#diff-2b0fff9a121c4bf67a04b4a82b44d2f2a5cc48a69d8c299b46072978d7cf1212L20